docs/fast_data/configuration/config_maps/erSchema.md

```diff
@@ -167,6 +171,140 @@ Example:
+
+You can always delete any projection or condition already configured. You have two ways of doing this.
+
+You can do it by selecting the projection box or the condition line you want to remove and pressing the _backspace_ or the _Canc_ key.
+
+Otherwise, you can select a projection or a condition and click on the _Delete_ button in the bottom right position of the right side panel.
+
+:::caution
+Removing a projection will also delete all of the conditions starting and ending from it.
+:::
+
+### Interact with the elements in the Canvas
+
+:::caution
+The position of the elements (boxes and lines) in the canvas is automatically calculated to give the best overview of the ER Schema. You are free to reorganize projections in the canvas but these updates will not be saved when you will commit your changes.
+:::
+
+The canvas is made to help you create and improve your ER Schema and also to give you a look at the whole picture of the Single View that will be created from it. To help you to have the full picture of it, there are several buttons on the bottom-left side of the canvas to adjust the zoom, to lock the canvas to move around without the risk of moving boxes and lines, and an _Autolayout_ button that restores the canvas elements to the best position for a complete view. 
+
+You can also zoom in and out with the scroll wheel of the mouse.
+
+Also, in case you make any mistake, you can use the _Undo_ and _Redo_ functionality: they can be toggled via the button at the bottom-right side of the canvas, or by pressing _Ctrl+Z__ and _Ctrl+Y_. 
+
+### The ER Schema code panel
+
+Any update on the canvas will be translated into the ER Schema in real-time, and you can review this with the left side panel of the canvas.
+
+This panel, visible by clicking on the `</>` icon, opens and closes a Code Editor which is fully working and allows you to see the structure of the ER Schema while you are updating it. This JSON structure will be the same structure that will be used in the Fast Data microservices. 
+
+It is also possible to interact with the content of the Code Editor, like adding or removing lines. Updating the ER Schema from the Code Editor will reflect automatically in the canvas.
+
+Also, the Code Panel allows you to copy and paste your schema into an external file. You might want to use this feature to test the ER Schema with the [Fast Data Low Code Test Template application](https://github.com/mia-platform/fast-data-low-code-test-template). You can also export the content to the ER Schema Config Map used by the [Real Time Updater](/fast_data/configuration/realtime_updater/low_code.md#er-schema-configuration) and the [Single View Trigger Generator](/fast_data/configuration/single_view_trigger_generator.md), or you can just use it to create another ER Schema as well.
+
+### Error management
+
+The No Code feature can identify the most common problems in the ER Schema and it can prevent further modifications until these problems are not solved.
```

Comment:
```suggestion
The No Code feature can identify the most common problems in the ER Schema and it can prevent further modifications until these problems are solved.
```